enterprise/admin/views.py

```diff
@@ -45,6 +45,8 @@
 )
 from enterprise.utils import get_reversed_url_by_site, send_email_notification_message
 
+from openedx.core.djangoapps.site_configuration import helpers as configuration_helpers
```

Comment:
Take a look at how we import edx-platform modules in other files:

https://github.com/edx/edx-enterprise/blob/master/enterprise/views.py#L52
https://github.com/edx/edx-enterprise/blob/master/enterprise/views.py#L66
https://github.com/edx/edx-enterprise/blob/master/enterprise/views.py#L424

This allows us to use edx-platform code in the LMS runtime, but also be able to mock calls to those modules in the edx-enterprise tests.

Reply:
Ahh, so the tests don't pull in the rest of edx-platform.. Didn't realize. Good to know, will update!